near/fast-auth-signer
# feat: add support for docker container envs by supporting runtime env variables

Support environment variable injection at compile-time and runtime. This is required so that our docker image can be run with custom environment variables which currently does not work since our image is already bundled with env variables that were passed at build time.

Testing Instructions:
- Build and run the Docker container.
- Pass necessary environment variables at runtime (e.g. using `-e` flag).
- Confirm that you are able to override defaults provided by webpack and at build time
